packages/pangea-sdk/tests/integration/test_authn.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2022 Pangea Cyber Corporation
+# Author: Pangea Cyber Corporation
+
+import random
+import unittest
+
+import pangea.exceptions as pexc
+from pangea import PangeaConfig
+from pangea.services.authn.authn import AuthN
+from pangea.services.authn.models import IDProvider
+from pangea.tools_util import TestEnvironment, get_test_domain, get_test_token
+
+TEST_ENVIRONMENT = TestEnvironment.DEVELOP
+
+
+RANDOM_VALUE = random.randint(0, 10000000)
+EMAIL_TEST = f"andres.tournour+test{RANDOM_VALUE}@pangea.cloud"
+EMAIL_DELETE = f"andres.tournour+delete{RANDOM_VALUE}@pangea.cloud"
+EMAIL_INVITE = f"andres.tournour+invite{RANDOM_VALUE}@pangea.cloud"
+PASSWORD_OLD = "My1s+Password"
+PASSWORD_NEW = "My1s+Password_new"
+PROFILE_OLD = {"name": "User name", "country": "Argentina"}
+PROFILE_NEW = {"age": "18"}
+USER_IDENTITY = None  # Will be set once user is created
+
+# tests that should be run in order are named with <letter><number>. Letter to make tests groups and number to order them inside that group
+
+
+def print_api_error(e: pexc.PangeaAPIException):
+    print(e)
+    for ef in e.errors:
```

Comment:
what is ef?

Reply:
errors is a list of error field, I'll change that var name. Maybe it's a little bit more "GO-dantic"